bcipy/helpers/visualization.py

```diff
@@ -119,35 +117,6 @@ def generate_offline_analysis_screen(
 
     fig_handles.append(fig)
 
-    if plot_lik_dens:
```

Comment:
I'd like to discuss this one to understand who is currently using this figure and how (and whether they would be OK with removing it). 

If you'd prefer a written description, I'd be happy to explain my thinking that way, but it might be easier to discuss in person. Just let me know which you'd prefer, and let's keep talking about this item

Reply:
We discussed and decided to:
1. remove for now, since the error cases covered by this figure are pretty well covered by tests now
2. come up with other debugging figures and metrics for future use, that work for any `SignalModel`

Reply:
https://www.pivotaltracker.com/story/show/178045928